pkgs/by-name/re/re-isearch/package.nix

```
@@ -47,14 +50,23 @@ stdenv.mkDerivation {
   preInstall = ''
     mkdir -p $out/{bin,lib}
   '';
-  postInstall = ''
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,lib}
+
+    cp ../bin/{Iindex,Isearch,Iutil,Idelete,zpresent,Iwatch,zipper} $out/bin
     cp ../lib/*.so $out/lib/
+
+    runHook postInstall
   '';
 
-  meta = with lib; {
+  meta = {
     description = "Novel multimodal search and retrieval engine";
     homepage = "https://github.com/re-Isearch/";
```

```suggestion
    homepage = "https://nlnet.nl/project/Re-iSearch/";
```
Seems more useful than the very bare-bones github org